src/hitchhiker/konserve.cljc

```diff
@@ -0,0 +1,162 @@
+(ns hitchhiker.konserve
+  (:refer-clojure :exclude [resolve subvec])
+  (:require [clojure.core.rrb-vector :refer [catvec subvec]]
+            #?(:clj [clojure.core.async :refer [chan promise-chan put!] :as async]
+               :cljs [cljs.core.async :refer [chan promise-chan put!] :as async])
+            [konserve.core :as k]
+            [konserve.memory :refer [new-mem-store]]
+            [hasch.core :refer [uuid]]
+            [clojure.set :as set]
+            #?(:clj [hitchhiker.tree.core :refer [go-try <?] :as core]
+               :cljs [hitchhiker.tree.core :as core])
+            [hitchhiker.tree.messaging :as msg])
+  #?(:cljs (:require-macros [hitchhiker.tree.core :refer [go-try <?]])))
+
```

Comment:
That certainly makes sense, but I actually mean something different: the structures that the tree uses for pointers (such as `delay`) aren't serializable. Could you document how the tree is stored in the serialized form? If you could cover things like which EDN lib is doing the serialization, what type are node keys, etc.